mpyg321/MPyg123Player.py

```diff
@@ -20,7 +20,7 @@ def __init__(
         super().__init__(player, audiodevice, performance_mode, custom_args)
         if performance_mode:
             self.silence_mpyg_output()
-        self.is_muted = False            
+        self.__is_muted = False            
 
     def process_output_ext(self, action):
         """Processes specific output for mpg123 player"""
```

Comment:
Can you please set `self._is_muted` to the right values when triggering the events? For consistency.